docs/sources/flow/reference/cli/convert.md

```diff
@@ -30,7 +30,7 @@ The following flags are supported:
 
 * `--report`, `-r`: The filepath and filename where the report is written.
 
-* `--source-format`, `-f`: Required. The format of the source file. Supported formats: [prometheus].
+* `--source-format`, `-f`: Required. The format of the source file. Supported formats: [prometheus], [promtail].
```

Comment:
I think the intent of the `[]` syntax was to be:

```suggestion
* `--source-format`, `-f`: Required. The format of the source file. Supported formats: [prometheus, promtail].
```

Reply:
it was meant to be a link to an anchor, but indeed something was wrong - I forgot to add the link reference.

Reply:
Ah, my bad, thanks!